core/genesis.go

```diff
@@ -80,6 +73,7 @@ type Genesis struct {
 	Alloc         GenesisAlloc        `json:"alloc"      gencodec:"required"`
 	AirdropHash   common.Hash         `json:"airdropHash"`
 	AirdropAmount *big.Int            `json:"airdropAmount"`
+	AirdropData   []byte              `json:"-"` // provided in a separate file, not serialized in this struct.
```

Comment:
does that mean we need to restart our WAGMI nodes with the new configuration? Can we also remove the json file in this PR?

Reply:
I think we need to provide a configuration for wagmi nodes in any way (even if we keep it in the subnet-evm repo). So it's probably best to move them to a correct place (wagmi folder under `public-chain-assets`).